siquery_cli/src/main.rs

```diff
@@ -89,12 +89,12 @@ fn main() {
                 println!("----------------------------------------------------------------");
             }
             #[cfg(not(windows))]
-            println!("Not implemented!");
+                println!("Not implemented!");
         }
 
         "wmi_services" => {
             #[cfg(target_os = "windows")]
-            for service in &system_info.wmi_services {
+                for service in &system_info.wmi_services {
```

Comment:
why was this indentation changed? code inside the { } scope should be indented one lever deeper if this is changed.